mvc/controllers.js

```diff
@@ -10,8 +10,14 @@ exports.getTopicsController = (req, res, next) => {
         .then((topics) => {
             res.status(200).send({ topics: topics })
         })
-        .catch((err)=>{
-            console.log('Error Caught: ', err)
-            next(err)
-        })
+        .catch(next)
+}
+
+exports.getArticlesController = (req, res, next) => {
```

Comment:
Just be careful with your naming here. As this is getting an individual article, I'd call this `getArticle` rather than `getArticles`, it'll just ensure there's no confusion when you move on to the next task 👍